cli/functionary/config.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+
+import click
+from dotenv import get_key, set_key
+
+
+def save_config_value(key, value):
+    functionary_dir = Path.home() / ".functionary"
+    if not functionary_dir.exists():
+        functionary_dir.mkdir()
+
+    config_file = functionary_dir / "config"
+    set_key(
+        config_file,
+        key,
+        value,
+        quote_mode="always",
+        export=False,
+        encoding="utf-8",
+    )
+
+
+def get_config_value(key):
+    try:
+        config_file = Path.home() / ".functionary" / "config"
+        value = get_key(config_file, key)
+        return value
+    # if path not found or key not found, raise error
+    except KeyError:
```

Comment:
With the changes you made, I don't believe KeyError will ever be thrown here.

`get_key` returns `None` if the provided file doesn't exist or the key is not present in the file.  You'll want to handle that, otherwise you'll be returning `None` and something will blow up downstream because it was expecting a string to come back.

Also, if the config file is present, but can't be read, it looks like a `PermissionError` gets thrown, so you'll want to handle that (both here and in the set function) so that you can return a nice error rather than just spewing a stack trace to the screen.